src/lib/db/feature-strategy-store.ts

```diff
@@ -345,12 +386,28 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
                 'feature_environments.environment',
                 'environments.name',
             );
+
+        if (this.flagResolver.isEnabled('toggleTagFiltering')) {
+            query = query.leftJoin(
+                'feature_tag as ft',
+                'ft.feature_name',
+                'features.name',
+            );
+        }
+
+        const rows = await query;
+
         if (rows.length > 0) {
             const overview = rows.reduce((acc, r) => {
                 if (acc[r.feature_name] !== undefined) {
                     acc[r.feature_name].environments.push(
                         FeatureStrategiesStore.getEnvironment(r),
                     );
+                    if (this.isUnseenTag(acc[r.feature_name], r)) {
```

Comment:
why do it need to be unseenTag?

Reply:
See: https://github.com/Unleash/unleash/pull/2396#issuecomment-1313539524